### DISCUSSION TITLE
仓辉无法正常使用

### DISCUSSION THREAD

**Discussion by uepaz** [yatori-dev/yatori-go-console#20] [Q&A] [answered]
报错：
10月 30, 2024 2:00:04 下午 okhttp3.internal.platform.Platform log
警告: A connection to （仓辉网址省略） was leaked. Did you forget to close a response body? To see where this was allocated, set the OkHttpClient logger level to FINE: Logger.getLogger(OkHttpClient.class.getName()).setLevel(Level.FINE);
下面是配置（具体信息省略）
![屏幕截图 2024-10-30 140254](https://github.com/user-attachments/assets/703dc95c-03ca-4225-b17d-8f1b4817e6c6)

**Comment by Changbaiqi** [comment, marked as answer]
看了下链接，这个本质是英华套壳并不是真正的仓辉，所以只需要把CANGHUI改成YINGHUA即可。

**Reply by uepaz** [reply]
好的，感谢
